docs/docs/docs/getting-started/installation.md

```diff
@@ -138,8 +142,16 @@ You now need to setup the environment. This follows next.
 
 #### Development Setup
 
+**Note:** If you are a developer looking to contribute to this project, follow the guidelines below.
+
+If you want to install and use Talawa Admin for your organization, please refer to the Production Setup section instead.
+
 If you prefer to use Docker, you can install the app using the following command:
 
+**Important:** Before running any Docker commands, ensure that the Docker daemon is running.
+- On Windows/macOS: Open Docker Desktop and verify it is running.
```

Comment:
1. Remove this section. It is not necessary.
2. Add a bullet in the  docker prerequisite section that docker needs to be running.